.rubocop.yml

```diff
@@ -543,6 +543,3 @@ Style/FormatStringToken:
 # are located.
 RSpec/FilePath:
   Enabled: false
-
```

Comment:
@ekohl This is somehow related to our current build issues.

Reply:
Yes. You either need to write the tests in the correct style or keep this in. For now I'd recommend to keep this and rewrite in a separate PR.

Reply:
I have to go to kindergarten now. It would be great if you could define an override in the .sync.yml.

Reply:
The rubocop template doesn't have an option for that and in the short term I'd just be lazy and modify it manually as has been done for the longest time in this module.